addons/manifests/cni-webhook-manifests.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
+  namespace: tkg-system
+spec:
+  type: ExternalName
+  externalName: 127.0.0.1
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  labels:
+    webhook-cert: self-managed
```

Comment:
This label needs to match updated label here and other places. Just the key is sufficient, value can be empty.

Reply:
Fixed